config/samples/dataplane_v1beta1_openstackdataplanedeployment_post_ceph_hci.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneDeployment
+metadata:
+  name: deployment-post-ceph
+spec:
+  nodeSets:
+    - openstack-edpm-hci
+  # Create this deployment after Ceph is deployed
+  # on EDPM nodes in an HCI scenario. Create a
+  # nova-custom-ceph service which uses a ConfigMap
+  # containing libvirt overrides for Ceph RBD.
+  servicesOverride:
+    - ceph-client
+    - ovn
+    - libvirt
+    - nova-custom-ceph
```

Comment:
Just a heads-up -- there is already a proposal to rename `servicesOverride` to just `services`: https://github.com/openstack-k8s-operators/dataplane-operator/pull/465

We'll have to refactor the VA1 data plane CRs to accommodate the proposed changes as well

Reply:
> Just a heads-up -- there is already a proposal to rename `servicesOverride` to just `services`: #465

Thanks for pointing that out. I think the review of #465 might take some time to get resolved. I'd be happy to send in a follow up PR after #465 merges. I'd rather merge this to promote the new pattern for HCI with what works today and not block on if it will be called `services` or `servicesOverride`. If the name changes it's an easy fix I will take care of right away. Thus, I request this not be blocked by #465. 

> We'll have to refactor the VA1 data plane CRs to accommodate the proposed changes as well

I'll be sending a PR to https://github.com/openstack-k8s-operators/architecture to update VA1 today.

Reply:
> We'll have to refactor the VA1 data plane CRs to accommodate the proposed changes as well

https://github.com/openstack-k8s-operators/architecture/pull/12